src/app/components/about-me-page/about-me-page.component.ts

```diff
@@ -1,15 +1,44 @@
-import { Component, OnInit } from '@angular/core';
-
+import { Component, ElementRef, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { CarouselModule } from 'ngx-owl-carousel-o';
 @Component({
   selector: 'app-about-me-page',
   templateUrl: './about-me-page.component.html',
   styleUrls: ['./about-me-page.component.scss']
 })
 export class AboutMePageComponent implements OnInit {
+  @ViewChild('journeyItems') journeyItemsRef: any
+  public journeyCurrentIndex = 1;
 
   constructor() { }
 
   ngOnInit(): void {
   }
 
+  journeyNavigatePrevious() {
```

Comment:
You should set the access scope and return type on your functions

Reply:
Done